src/controllers/EventDetailController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace controllers;
+
+use models\Booking;
+use models\Event;
+use models\User;
+
+class EventDetailController extends Controller
+{
+    public function render($params)
+    {
+        if (isset($_GET['event_id'])) {
+            $event = Event::getInstance();
+            $eventById = $event->getEventById(htmlspecialchars($_GET['event_id']));
+
+            $user = User::getInstance();
+            if (isset($eventById->creator_id)) {
+                $creator = $user->getUserById($eventById->creator_id);
+            }
+            if (empty($creator)) {
+                $creator = "";
+            }
+            $eventById->creator = $creator->username;
```

Comment:
```suggestion
            $eventById->$creator = $creator->username;
```
or
```suggestion
            $eventById->creator_id = $creator->username;
```
Does $eventById have a 'creator' field?